tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6637,9 +6661,17 @@ Status ConvertResize(OpConverterParams* params) {
 
   // Set layer parameters.
   layer->setResizeMode(resize_mode);
-  layer->setOutputDimensions(output_dimensions);
   layer->setAlignCorners(align_corners);
 
+  // Set output shape.
+  if (params->use_implicit_batch || HasStaticShape(inputs_dims)) {
+    // If all shape dims are known at build time, pass them as a parameter.
+    layer->setOutputDimensions(output_shape_dims);
+  } else {
+    // Otherwise, the output shape tensor becomes an extra input to the layer.
```

Comment:
Let's revise the comment:
Otherwise, pass the output shape tensor to the resize layer as an input.